src/SpectatorView.Unity/Assets/StreamingAssets/HiddenARCore.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: 90a1ed2970cd6564dbefade7e5ad0260
+folderAsset: yes
+DefaultImporter:
+  externalObjects: {}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
```

Comment:
This seems like it shouldn't be added, as I see no content in the PR for this folder

Reply:
yeah sounds good